src/components/shared/HeaderBackground.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  Text,
+  StyleSheet,
+} from 'react-native';
+
+import LinearGradient from 'react-native-linear-gradient';
+
+import { Colors } from '../../Colors';
+import GlobalStyles from '../../GlobalStyles';
+
+export default (props) => {
+  const styles = StyleSheet.create({  
```

Comment:
Leaving a comment here to standardize placement for `StyleSheet` in a markdown. I'll do this in a follow on Issue.

Reply:
Addressed in #12 (low priority)